src/vs/editor/contrib/codelens/codelensController.ts

```diff
@@ -402,6 +403,62 @@ export class CodeLensContribution implements IEditorContribution {
 			}
 		});
 	}
+
+	public getLenses(): CodeLensWidget[] {
+		return this._lenses;
+	}
+}
+
+export class ShowLensesInCurrentLineCommand extends EditorCommand {
+	public runEditorCommand(accessor: ServicesAccessor, editor: ICodeEditor, args: any): void | Promise<void> {
+		const quickInputService = accessor.get(IQuickInputService);
+		const commandService = accessor.get(ICommandService);
+		const notificationService = accessor.get(INotificationService);
+
+		const lineNumber = editor.getSelection()?.positionLineNumber;
+		const codelensController = editor.getContribution(CodeLensContribution.ID) as CodeLensContribution;
+
+		const activeLensesWidgets = codelensController.getLenses().filter(lens => lens.getLineNumber() === lineNumber);
+
+		const commandArguments: Map<string, any[] | undefined> = new Map();
+
+		const picker = quickInputService.createQuickPick();
```

Comment:
You can use [`IQuickInputService#pick`](https://github.com/microsoft/vscode/blob/527513a911d5f2ba44d599d847f9c63c6fce178c/src/vs/platform/quickinput/common/quickInput.ts#L36) instead of `createQuickPick`. For simple cases like this you don't to write that much code

Reply:
Nice! Just updated it! Thanks @jrieken